arraycontext/impl/pytato/compile.py

```diff
@@ -40,13 +40,25 @@
 import pyopencl.array as cla
 import pytato as pt
 import itertools
+from pytools.tag import Tag
 
 from pytools import ProcessLogger
 
 import logging
 logger = logging.getLogger(__name__)
 
 
+class FromActxCompile(Tag):
```

Comment:
Sorry, spotted another thing: This should be hooked into the docs.

Reply:
`FromArrayContextCompile`? :grin: